src/renderer/components/message/Message.tsx

```diff
@@ -189,6 +189,9 @@ function buildContextMenu(
   const showAttachmentOptions = !!message.file && !message.isSetupmessage
   const showCopyImage = !!message.file && message.viewType === 'Image'
 
+  const showResend =
+    message.viewType === 'Webxdc' && message.sender.id === C.DC_CONTACT_ID_SELF
```

Comment:
As, according to #3259, the goal is an option to resend own message, webxdc or not, isn't this boolean expression incorrect?

Reply:
no it is correct as the discussion there yielded no clear results it is better to use copy the android behaviour for now.
But it is very easy to change this in the future if we should decide otherwise.
another idea would be to also show it on failed messages that failed to send - but I still think we need to discuss this also with more people.